artifactory/utils/container/image.go

```diff
@@ -1,49 +1,52 @@
 package container
 
 import (
+	"net/http"
 	"path"
 	"strings"
 
+	"github.com/jfrog/jfrog-client-go/artifactory"
 	"github.com/jfrog/jfrog-client-go/utils/errorutils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 )
 
 type Image struct {
-	// Image name includes the registry domain, image base name and image tag e.g.: https://my-registry/docker-local/hello-world:latest.
+	// Image name includes the registry domain, image base name and image tag e.g.: my-registry:port/docker-local/hello-world:latest.
 	name string
 }
 
+func NewImage(imageTag string) *Image {
+	return &Image{name: imageTag}
+}
+
 // Get image name
 func (image *Image) Name() string {
 	return image.name
 }
 
-// Get image relative path in Artifactory.
-func (image *Image) GetPath() (string, error) {
+// Get image name from tag by removing the prefixed registry hostname.
```

Comment:
I suggest we add an example to this description, showing the tag, and the resulting image name.